applications/sintering/include/pf-applications/sintering/postprocessors.h

```diff
@@ -1180,11 +1180,10 @@ namespace Sintering
       run_flooding(const typename DoFHandler<dim>::cell_iterator &cell,
                    const BlockVectorType &                        solution,
                    LinearAlgebra::distributed::Vector<Number> &   particle_ids,
-                   const unsigned int                             id)
+                   const unsigned int                             id,
+                   const double threshold_lower     = 0.5,
```

Comment:
This is a different value!?

Reply:
I was experimenting with different values, but 0.8 indeed looks optimal, I set it back.